### PR TITLE
[CTOR-614] refresh Keepalived SNMP doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 1]
```diff
@@ -2,48 +2,332 @@
...
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-pack-applications-keepalived-snmp
+```
+
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:93]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 2]
```diff
@@ -2,48 +2,332 @@
...
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-plugin-Applications-Keepalived-Snmp
+```
+
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:138]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 3]
```diff
@@ -2,48 +2,332 @@
...
+	--snmp-community='my-snmp-community'  \
+	--warning-status='' \
+	--critical-status='' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All VRRP instances are ok | '*vrrp*#status'=;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:213]: '*vrrp*#status'=;;;; -> rien du tout?

File: pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 4]
```diff
@@ -2,48 +2,331 @@
...
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-pack-applications-keepalived-snmp
+```
+
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:92]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 5]
```diff
@@ -2,48 +2,331 @@
...
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-plugin-Applications-Keepalived-Snmp
+```
+
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:139]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 6]
```diff
@@ -2,48 +2,331 @@
...
+	--snmp-community='my-snmp-community'  \
+	--warning-status='' \
+	--critical-status='' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All VRRP instances are ok | '*vrrp*#status'=;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:212]: same

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md [hunk 7]
```diff
@@ -2,48 +2,332 @@
...
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                        |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **App-Keepalived-SNMP-custom** host template |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-keepalived-snmp.md:37]:
```suggestion
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/applications-protocol-snmp/) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **App-Keepalived-SNMP-custom** |
```

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**